synfig-studio/src/gui/canvasview.h

```diff
@@ -242,6 +243,8 @@ class CanvasView : public Dockable, public etl::shared_object
 private:
 	WorkArea *work_area;
 
+	Widget_Timetrack* current_widget_timetrack;
+
```

Comment:
You don't need this.
`CanvasView` has a method `get_ext_widget()`. Get the timetrack calling it with "timetrack" argument.

The dock registers the widget in the method `Dock_Timetrack2::init_canvas_view_vfunc(etl::loose_handle<CanvasView> canvas_view)`

Reply:
sounds perfect, I saw the method but didn't see the definition xD.